src/gluonts/model/deepstate/_estimator.py

```diff
@@ -174,6 +181,7 @@ def __init__(
         noise_std_bounds: ParameterBounds = ParameterBounds(1e-6, 1.0),
         prior_cov_bounds: ParameterBounds = ParameterBounds(1e-6, 1.0),
         innovation_bounds: ParameterBounds = ParameterBounds(1e-6, 0.01),
+        imputation_method: str = "standard",
```

Comment:
So maybe we should remove this string-based selection. I would say we only modify the DeepAREstimator for now, using the MissingValueImputation constructor argument, and then we go about updating the rest